dockstore-webservice/src/main/resources/logstash/logsToElastic.conf

```diff
@@ -0,0 +1,25 @@
+input {
+  tcp {
+    port => 5044
+    codec => json
+  }
+  file {
+    path => "/tmp/access_log"
+    start_position => "beginning"
+  }
+}
+
+
+filter {
+  grok {
+    match => { "message" => "%{COMBINEDAPACHELOG}" }
+  }
+  date {
+    match => [ "timestamp" , "dd/MMM/yyyy:HH:mm:ss Z" ]
+  }
+}
+
+output {
+  elasticsearch { hosts => ["172.17.0.2:9200"] }
```

Comment:
I don't recognize this IP address. Is this part of an incomplete test?
This seems like it should be configured from the main dropwizard config either way

Reply:
I added that as an example, but I suppose I can remove it since it's not used anywhere here.  Also, it's in the primer already.